src/klein/test/test_resource.py

```diff
@@ -1133,9 +1133,11 @@ def test_urlDecodeErrorRepr(self) -> None:
         """
         URLDecodeError.__repr__ formats properly.
         """
+        error = _URLDecodeError([("VALUE", ValueError), ("TYPE", TypeError)])
         self.assertEqual(
-            "<URLDecodeError(errors=<class 'ValueError'>)>",
-            repr(_URLDecodeError(ValueError)),
+            "<URLDecodeError(errors=[('VALUE', <class 'ValueError'>), "
+            "('TYPE', <class 'TypeError'>)])>",
+            repr(error),
```

Comment:
This test was wrong insofar as it passed an invalid argument to `_URLDecodeError`.